ext/iconv/tests/eucjp2iso2022jp.phpt

```diff
@@ -11,7 +11,7 @@ error_reporting=2039
 function hexdump($str) {
     $len = strlen($str);
     for ($i = 0; $i < $len; ++$i) {
-        printf("%02x", ord($str{$i}));
+        printf("%02x", ord($str[$i]));
```

Comment:
@nikic seems this doesn't generate any deprecation warning ...

Reply:
It looks like the test sets error_reporting, probably it's suppressed.